src/Mvc/Mvc.ViewFeatures/src/Controller.cs

```diff
@@ -46,7 +48,7 @@ public ViewDataDictionary ViewData
                     _viewData = new ViewDataDictionary(new EmptyModelMetadataProvider(), ControllerContext.ModelState);
                 }
 
-                return _viewData;
+                return _viewData!;
```

Comment:
`??=`?

Reply:
I guess. Although the if block has a nice comment which says it's only ever run inside a unit test. I'd prefer to keep it that way.